===================================================================
RCS file: /cvsroot/shadow/libmisc/log.c,v
retrieving revision 1.7
retrieving revision 1.8
diff -u -r1.7 -r1.8
--- shadow/libmisc/log.c
+++ shadow/libmisc/log.c
@@ -89,6 +89,7 @@
 
 	ll_time = newlog.ll_time;
 	time (&ll_time);
+	newlog.ll_time = ll_time;
 	strncpy (newlog.ll_line, line, sizeof newlog.ll_line);
 #if HAVE_LL_HOST
 	strncpy (newlog.ll_host, host, sizeof newlog.ll_host);
